components/Ink/src/MDCInkTouchController.h

```diff
@@ -57,6 +57,14 @@
  */
 @property(nonatomic, assign) BOOL cancelsOnDragOut;
 
+/**
+ If enabled, the ink gesture will require failure of UIScrollView gesture recognizers in order to
+ activate.
+
+ Defaults to NO.
+ */
+@property(nonatomic, assign) BOOL requiresFailureOfScrollViewGestures;
```

Comment:
I don't think the new naming and documentation is clear for me as a client.

When it is set to YES it doesn't require failure of UIScrollView gesture recognizers. The UIScrollView gesture recognizers aren't the ones who are failing, but rather the MDCInkGestureRecognizer is failing if there are any gestures of the UIScrollView that aren't taps or long presses.

Reply:
@jverkoey ^

Reply:
That's the opposite of what this method does.

When this property is enabled, `shouldRequireFailureOfGestureRecognizer: otherGestureRecognizer ` will be able to return YES, indicating that `otherGestureRecognizer ` should fail prior to `gestureRecognizer` being able to enter `UIGestureRecognizerStateBegan`.

Reply:
ah sorry, my bad here. I thought we were talking about `gestureRecognizer:shouldBeRequiredToFailByGestureRecognizer:`. 
Thanks for clarifying

Reply:
All good! The gesture recognizer state machine's got a good amount going on.